### PR TITLE
ls, mv, rm: add Spanish translations

### PR DESCRIPTION
**Pull request by nicomt**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/common/mv.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# mv
+
+> Mover o renombra archivos y directorios.
```
Comment (navarroaxel) [inline pages.es/common/mv.md:3]:
```suggestion
> Mueve o renombra archivos y directorios.
```

File: pages.es/common/mv.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Mover archivos en ubicaciones arbitrarias:
```
Comment (navarroaxel) [inline pages.es/common/mv.md:5]: Entiendo que hay que usar el imperativo como `lista` en `ls`:
```suggestion
- Mueve archivos en ubicaciones arbitrarias:
```

File: pages.es/common/mv.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`mv {{origen}} {{destino}}`
+
+- Mover sin solicitar confirmación antes de sobrescribir archivos existentes:
+
+`mv -f {{origen}} {{destino}}`
+
+- Solicitar confirmación antes de sobrescribir archivos existentes, independientemente de los permisos del archivo:
```
Comment (navarroaxel) [inline pages.es/common/mv.md:13]:
```suggestion
- Solicita confirmación antes de sobrescribir archivos existentes, independientemente de los permisos del archivo:
```

File: pages.es/common/mv.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+`mv -i {{origen}} {{destino}}`
+
+- No sobrescribir archivos existentes en el destino:
+
+`mv -n {{origen}} {{destino}}`
+
+- Mover archivos en modo detallado, mostrando los archivos después de moverlos:
```
Comment (navarroaxel) [inline pages.es/common/mv.md:21]:
```suggestion
- Mueve archivos en modo detallado, mostrando los archivos después de moverlos:
```

File: pages.es/common/mv.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+- Mover sin solicitar confirmación antes de sobrescribir archivos existentes:
+
+`mv -f {{origen}} {{destino}}`
+
+- Solicitar confirmación antes de sobrescribir archivos existentes, independientemente de los permisos del archivo:
+
+`mv -i {{origen}} {{destino}}`
+
+- No sobrescribir archivos existentes en el destino:
```
Comment (navarroaxel) [inline pages.es/common/mv.md:17]:
```suggestion
- No sobrescribe archivos existentes en el destino:
```

File: pages.es/common/mv.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# mv
+
+> Mover o renombra archivos y directorios.
+
+- Mover archivos en ubicaciones arbitrarias:
+
+`mv {{origen}} {{destino}}`
+
+- Mover sin solicitar confirmación antes de sobrescribir archivos existentes:
```
Comment (navarroaxel) [inline pages.es/common/mv.md:9]: I don't like the example descriptions for the English page:
```suggestion
- Mueve sin solicitar confirmación antes de sobrescribir archivos existentes:
```

File: pages.es/common/rm.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# rm
+
+> Eliminar archivos o directorios.
```
Comment (navarroaxel) [inline pages.es/common/rm.md:3]:
```suggestion
> Elimina archivos o directorios.
```

File: pages.es/common/rm.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+- Elimina archivos de ubicaciones arbitrarias:
+
+`rm {{ruta/hacia/archivo}} {{ruta/hacia/otro/archivo}}`
```
Comment (navarroaxel) [inline pages.es/common/rm.md:7]: :thinking: Some Spanish pages use `ruta/hacia` but I prefer `ruta/al`, I think sounds a more complete sentence:
```suggestion
`rm {{ruta/al/archivo}} {{ruta/al/otro/archivo}}`
```

File: pages.es/common/rm.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+- Elimina, de forma recursiva, un directorio y todos sus subdirectorios:
+
+`rm -r {{ruta/hacia/directorio}}`
```
Comment (navarroaxel) [inline pages.es/common/rm.md:11]:
```suggestion
`rm -r {{ruta/al/directorio}}`
```

File: pages.es/common/rm.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+- Elimina un directorio a la fuerza, sin pedir confirmación ni mostrar mensajes de error:
+
+`rm -rf {{ruta/hacia/directorio}}`
```
Comment (navarroaxel) [inline pages.es/common/rm.md:15]:
```suggestion
`rm -rf {{ruta/al/directorio}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @nicomt, welcome and thanks for your submission. I left some comments.

**Comment by nicomt** [comment]
@navarroaxel Thanks for reviewing the pull request. I applied all your suggestions

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**